test/services/database/comment_upvote_repository_test.dart

```diff
@@ -0,0 +1,136 @@
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/database/comment/comment_id_firestore.dart";
+import "package:proxima/models/database/post/post_id_firestore.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+import "package:proxima/models/database/vote/upvote_state.dart";
+import "package:proxima/services/database/comment_repository_service.dart";
+import "package:proxima/services/database/comment_upvote_repository_service.dart";
+import "package:proxima/services/database/firestore_service.dart";
+import "package:proxima/services/database/post_repository_service.dart";
+import "package:proxima/services/database/upvote_repository_service.dart";
+
+import "../../mocks/data/comment_data.dart";
+import "../../mocks/data/firestore_user.dart";
+import "../../mocks/data/geopoint.dart";
+import "../../mocks/data/post_data.dart";
+
+void main() {
+  group("Testing comment upvote repository", () {
+    late FakeFirebaseFirestore firestore;
+    late PostRepositoryService postRepository;
+    late CommentRepositoryService commentRepository;
+    late UpvoteRepositoryService<CommentIdFirestore> commentUpvoteRepository;
+    late CommentDataGenerator commentDataGenerator;
+
+    late PostIdFirestore postId;
+    late CommentIdFirestore commentId;
+
+    setUp(() async {
+      firestore = FakeFirebaseFirestore();
+      postRepository = PostRepositoryService(firestore: firestore);
+      commentRepository = CommentRepositoryService(firestore: firestore);
+      commentDataGenerator = CommentDataGenerator();
+
+      final container = ProviderContainer(
+        overrides: [
+          firestoreProvider.overrideWithValue(firestore),
+          postRepositoryProvider.overrideWithValue(postRepository),
+          commentRepositoryProvider.overrideWithValue(commentRepository),
```

Comment:
I don't think those overrides are necessary. The code still works on my machine without them (the firestore override is sufficient)
```suggestion
```

Reply:
Totally right !
Fix in ade3a650aa88f5ca6b9ef4a7c1933710db9a9460.